jinja2/includes/translate_locales.html

```diff
@@ -0,0 +1,62 @@
+This template is automatically generated by scripts/translate_locales.py to make the languages name localizable. Do not edit it manually
```

Comment:
Oh sorry. The script was replaced by ``translate_locales_name`` management command.
I will update the line soon.